api/api.go

```diff
@@ -35,120 +30,115 @@ func HttpPOST(url string, data string) (resp *http.Response, err error) {
+		mux.GET("/renter/files", srv.renterFilesHandler)
+		mux.POST("/renter/load", srv.renterLoadHandler)
+		mux.POST("/renter/loadascii", srv.renterLoadAsciiHandler)
+		mux.POST("/renter/rename", srv.renterRenameHandler)
+		mux.POST("/renter/delete/*path", srv.renterDeleteHandler)
+		mux.GET("/renter/download/*path", srv.renterDownloadHandler)
+		mux.GET("/renter/share/*path", srv.renterShareHandler)
+		mux.GET("/renter/shareascii/*path", srv.renterShareAsciiHandler)
+		mux.POST("/renter/upload/*path", srv.renterUploadHandler)
 	}
 
 	// TransactionPool API Calls - Unfinished
 	if srv.tpool != nil {
-		srv.handleHTTPRequest(mux, "/transactionpool/transactions", srv.transactionpoolTransactionsHandler)
+		mux.GET("/transactionpool/transactions", srv.transactionpoolTransactionsHandler)
 	}
 
 	// Wallet API Calls
 	if srv.wallet != nil {
-		srv.handleHTTPRequest(mux, "/wallet", srv.walletHandler)                           // GET
-		srv.handleHTTPRequest(mux, "/wallet/address", srv.walletAddressHandler)            // GET
-		srv.handleHTTPRequest(mux, "/wallet/addresses", srv.walletAddressesHandler)        // GET
-		srv.handleHTTPRequest(mux, "/wallet/backup", srv.walletBackupHandler)              // GET
-		srv.handleHTTPRequest(mux, "/wallet/encrypt", srv.walletEncryptHandler)            // POST - COMPATv0.4.0
-		srv.handleHTTPRequest(mux, "/wallet/init", srv.walletInitHandler)                  // POST
-		srv.handleHTTPRequest(mux, "/wallet/load/033x", srv.walletLoad033xHandler)         // POST
-		srv.handleHTTPRequest(mux, "/wallet/load/seed", srv.walletLoadSeedHandler)         // POST
-		srv.handleHTTPRequest(mux, "/wallet/load/siag", srv.walletLoadSiagHandler)         // POST
-		srv.handleHTTPRequest(mux, "/wallet/lock", srv.walletLockHandler)                  // POST
-		srv.handleHTTPRequest(mux, "/wallet/seeds", srv.walletSeedsHandler)                // GET
-		srv.handleHTTPRequest(mux, "/wallet/siacoins", srv.walletSiacoinsHandler)          // POST
-		srv.handleHTTPRequest(mux, "/wallet/siafunds", srv.walletSiafundsHandler)          // POST
-		srv.handleHTTPRequest(mux, "/wallet/transaction/", srv.walletTransactionHandler)   // $(id) GET
-		srv.handleHTTPRequest(mux, "/wallet/transactions", srv.walletTransactionsHandler)  // GET
-		srv.handleHTTPRequest(mux, "/wallet/transactions/", srv.walletTransactionsHandler) // $(addr) GET
-		srv.handleHTTPRequest(mux, "/wallet/unlock", srv.walletUnlockHandler)              // POST
+		mux.GET("/wallet", srv.walletHandler)
+		mux.GET("/wallet/address", srv.walletAddressHandler)
+		mux.GET("/wallet/addresses", srv.walletAddressesHandler)
+		mux.GET("/wallet/backup", srv.walletBackupHandler)
+		mux.POST("/wallet/encrypt", srv.walletInitHandler) // COMPATv0.4.0
+		mux.POST("/wallet/init", srv.walletInitHandler)
+		mux.POST("/wallet/load/033x", srv.walletLoad033xHandler)
+		mux.POST("/wallet/load/seed", srv.walletLoadSeedHandler)
+		mux.POST("/wallet/load/siag", srv.walletLoadSiagHandler)
```

Comment:
For these 3 load calls, I'm wondering if the `load` is necessary in the path. `mux.POST("/wallet/033x")` seems pretty clear to me